ICAFIX/scripts/RecleanClassify.sh

```diff
@@ -0,0 +1,284 @@
+DeleteIntermediates="TRUE" #TRUE/FALSE
+
+CorticalLUT="$HCPPIPEDIR/global/config/FreeSurferCorticalLabelTableLut.txt"
+SubCorticalLUT="$HCPPIPEDIR/global/config/FreeSurferSubcorticalLabelTableLut.txt"
+
+if [ ! ${MRFixConcatName} = "" ] ; then
+    DropOutSubSTRING=""
+    for fMRIName in "${fMRINamesArray[@]}" ; do
+        if [ -e ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}_dropouts.nii.gz ] ; then
+        DropOutSubSTRING=`echo "${DropOutSubSTRING}${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}_dropouts.nii.gz "`
+        fi
+    done
+    if [ ! -z "${DropOutSubSTRING}" ] ; then
+        fslmerge -t ${StudyFolder}/${Subject}/MNINonLinear/Results/${MRFixConcatName}/${MRFixConcatName}_dropouts.nii.gz ${DropOutSubSTRING}
+        fslmaths ${StudyFolder}/${Subject}/MNINonLinear/Results/${MRFixConcatName}/${MRFixConcatName}_dropouts.nii.gz -Tmean ${StudyFolder}/${Subject}/MNINonLinear/Results/${MRFixConcatName}/${MRFixConcatName}_dropouts.nii.gz
+        WorkingDirectory="/tmp/${Subject}"
+        mkdir -p ${WorkingDirectory}
+        cp ${StudyFolder}/${Subject}/MNINonLinear/Results/${MRFixConcatName}/${MRFixConcatName}_dropouts.nii.gz ${WorkingDirectory}/${MRFixConcatName}_dropouts.nii.gz
+        gunzip -f ${WorkingDirectory}/${MRFixConcatName}_dropouts.nii.gz
+        "$HCPPIPEDIR"/global/scripts/MapVolumeToCIFTI.sh ${StudyFolder} ${Subject} ${MRFixConcatName} ${CorticalLUT} ${SubCorticalLUT} ${Caret7_Command} ${LowResMesh} ${RegName} ${SmoothingFWHM} ${FinalfMRIResolution} ${BrainOrdinatesResolution} ${WorkingDirectory}/${MRFixConcatName}_dropouts.nii ${StudyFolder}/${Subject}/MNINonLinear/Results/${MRFixConcatName}/${MRFixConcatName}_dropouts.dscalar.nii ${MRFixConcatName}_dropouts ${Flag} ${DeleteIntermediates} nii ${WorkingDirectory}
+        rm -r ${WorkingDirectory}
+    fi
+else
+    for fMRIName in "${fMRINamesArray[@]}" ; do
+        if [ -e ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}_dropouts.nii.gz ] ; then
+        WorkingDirectory="/tmp/${Subject}"
+        mkdir -p ${WorkingDirectory}
+        cp ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}_dropouts.nii.gz ${WorkingDirectory}/${fMRIName}_dropouts.nii.gz
+        gunzip -f ${WorkingDirectory}/${fMRIName}_dropouts.nii.gz
+        "$HCPPIPEDIR"/global/scripts/MapVolumeToCIFTI.sh ${StudyFolder} ${Subject} ${fMRIName} ${CorticalLUT} ${SubCorticalLUT} ${Caret7_Command} ${LowResMesh} ${RegName} ${SmoothingFWHM} ${FinalfMRIResolution} ${BrainOrdinatesResolution} ${WorkingDirectory}/${fMRIName}_dropouts.nii ${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}_dropouts.dscalar.nii ${fMRIName}_dropouts ${Flag} ${DeleteIntermediates} nii ${WorkingDirectory}
+        rm -r ${WorkingDirectory}
+        fi
+    done
+fi
+
+# compute additional features from matlab script
+#FIXME: parcellation files are saved under tICA folder--reclassify-as-signal-file
```

Comment:
This looks like a stray paste was added to this comment.

Why are they in the tICA folder?  You can just make a new folder in global/templates, or even use standard_mesh_atlases (or make a "templates" folder in ICAFIX, though it would be a first, and these parcellations aren't exactly specific to ICAFIX).

Reply:
Because they have been existed inside the tICA folder so I just use them instead of duplicating the same file in two different folders.